src/test/java/org/springframework/hateoas/LinkUnitTest.java

```diff
@@ -451,4 +452,17 @@ public void uriCreationRejectsLinkWithUnresolvedMandatoryParameters() {
 		assertThatExceptionOfType(IllegalStateException.class).isThrownBy(() -> new Link("/{segment}/path").toUri());
 	}
 
+	@Test
```

Comment:
Issue comment is missing.

Reply:
Hah hah! That is what one of our learned colleagues suggested I add in order to get the IDE to serve me the "run nullable inspections". It needs to be removed.